diff --git a/admin/inc/class.admin_asyncservice.inc.php b/admin/inc/class.admin_asyncservice.inc.php
index 21b1c6d4d1..39f7ffa3e7 100644
--- a/admin/inc/class.admin_asyncservice.inc.php
+++ b/admin/inc/class.admin_asyncservice.inc.php
@@ -99,16 +99,20 @@ class admin_asyncservice
 		{
 			Api\Config::save_value('asyncservice', $GLOBALS['egw_info']['server']['asyncservice']=$_POST['asyncservice'], 'phpgwapi');
 		}
+/*
 		if (!$async->only_fallback)
 		{
 			$installed = $async->installed();
 			if (is_array($installed) && isset($installed['cronline']))
 			{
+*/
 				$async_use['cron'] = lang('crontab only (recomended)');
+/*
 			}
 		}
 		$async_use['']    = lang('fallback (after each pageview)');
 		$async_use['off'] = lang('disabled (not recomended)');
+*/
 		echo '<p><b>'.htmlspecialchars(lang('Run Asynchronous services')).'</b>'.
 			' <select name="asyncservice" onChange="this.form.submit();">';
 		foreach ($async_use as $key => $label)
@@ -117,7 +121,7 @@ class admin_asyncservice
 			echo "<option value=\"$key\"$selected>".htmlspecialchars($label)."</option>\n";
 		}
 		echo "</select>\n";
-
+/*
 		if (is_array($installed) && isset($installed['cronline']))
 		{
 			echo ' &nbsp; <input type="submit" name="deinstall" value="'.htmlspecialchars(lang('Deinstall crontab'))."\">\n";
@@ -147,6 +151,8 @@ class admin_asyncservice
 			echo '<p><input type="submit" name="install" value="'.htmlspecialchars(lang('Install crontab'))."\">\n".
 				htmlspecialchars(lang("for the times below (empty values count as '*', all empty = every minute)"))."</p>\n";
 		}
+*/
+		echo '<p>'.htmlspecialchars(lang('Installed crontab')).": /etc/cron.d/egroupware.cron\n";
 
 		echo "<hr><table border=0><tr>\n";
 		foreach ($units as $u => $ulabel)
diff --git a/api/src/Egw.php b/api/src/Egw.php
index 8226e0c5dc..8542eb370c 100644
--- a/api/src/Egw.php
+++ b/api/src/Egw.php
@@ -641,12 +641,14 @@ class Egw extends Egw\Base
 					_egw_log_exception($ex);
 				}
 			}
+/* not needed as rpm uses /etc/cron.d/egroupware
 			// call the asyncservice check_run function if it is not explicitly set to cron-only
 			if (!$GLOBALS['egw_info']['server']['asyncservice'])	// is default
 			{
 				$async = new Asyncservice();
 				$async->check_run('fallback');
 			}
+*/
 			$this->db->disconnect();
 		}
 	}
