Index: egroupware/admin/inc/class.uiasyncservice.inc.php
===================================================================
--- egroupware/admin/inc/class.uiasyncservice.inc.php	(Revision 27137)
+++ egroupware/admin/inc/class.uiasyncservice.inc.php	(Arbeitskopie)
@@ -96,16 +96,20 @@
 			$config->save_repository();
 			unset($config);
 		}
+/*
 		if (!$async->only_fallback)
 		{
 			$installed = $async->installed();
 			if (is_array($installed) && isset($installed['cronline']))
 			{
+*/
 				$async_use['cron'] = lang('crontab only (recomended)');
+/*
 			}
 		}
 		$async_use['']    = lang('fallback (after each pageview)');
 		$async_use['off'] = lang('disabled (not recomended)');
+*/
 		echo '<p><b>'.lang('Run Asynchronous services').'</b>'.
 			' <select name="asyncservice" onChange="this.form.submit();">';
 		foreach ($async_use as $key => $label)
@@ -127,8 +131,9 @@
 		}
 		else
 		{
-			echo '<p>'.lang('Installed crontab').": \n";
+			echo '<p>'.lang('Installed crontab').": /etc/cron.d/egroupware\n";
 
+/*
 			if (is_array($installed) && isset($installed['cronline']))
 			{
 				echo "$installed[cronline]</p>";
@@ -143,6 +148,7 @@
 			}
 			echo '<p><input type="submit" name="install" value="'.lang('Install crontab')."\">\n".
 				lang("for the times below (empty values count as '*', all empty = every minute)")."</p>\n";
+*/
 		}
 
 		echo "<hr><table border=0><tr>\n";
