--- egroupware.orig/admin/inc/class.uiasyncservice.inc.php	2005-10-14 19:03:16.000000000 +0200
+++ egroupware/admin/inc/class.uiasyncservice.inc.php	2006-03-09 21:37:20.000000000 +0100
@@ -106,7 +106,7 @@
 
 			$last_run = $async->last_check_run();
 			$lr_date = $last_run['end'] ? $GLOBALS['egw']->common->show_date($last_run['end']) : lang('never');
-			echo '<p><b>'.lang('Async services last executed').'</b>: '.$lr_date.' ('.$last_run['run_by'].")</p>\n<hr>\n";
+			echo '<p><b>'. lang('Async services last executed'). '</b>: '. $lr_date. ' ('. $last_run['run_by']. ")</p>\n<hr>\n";
 
 			if (isset($_POST['asyncservice']) && $_POST['asyncservice'] != $GLOBALS['egw_info']['server']['asyncservice'])
 			{
@@ -116,16 +116,17 @@
 				$config->save_repository();
 				unset($config);
 			}
-			if (!$async->only_fallback)
+
+			/*if (!$async->only_fallback)
 			{
 				$installed = $async->installed();
 				if (is_array($installed) && isset($installed['cronline']))
-				{
+				{*/
 					$async_use['cron'] = lang('crontab only (recomended)');
-				}
+			/*	}
 			}
 			$async_use['']    = lang('fallback (after each pageview)');
-			$async_use['off'] = lang('disabled (not recomended)');
+			$async_use['off'] = lang('disabled (not recomended)'); */
 			echo '<p><b>'.lang('Run Asynchronous services').'</b>'.
 				' <select name="asyncservice" onChange="this.form.submit();">';
 			foreach ($async_use as $key => $label)
@@ -147,8 +148,9 @@
 			}
 			else
 			{
-				echo '<p>'.lang('Installed crontab').": \n";
+				echo '<p>'.lang('Installed crontab').": /etc/cron.d/egroupware\n";
 
+				/*
 				if (is_array($installed) && isset($installed['cronline']))
 				{
 					echo "$installed[cronline]</p>";
@@ -161,6 +163,7 @@
 				{
 					echo '<b>'.lang('asyncservices not yet installed or other error (%1) !!!',$installed['error'])."</b></p>\n";
 				}
+				*/
 				echo '<p><input type="submit" name="install" value="'.lang('Install crontab')."\">\n".
 					lang("for the times below (empty values count as '*', all empty = every minute)")."</p>\n";
 			}
